src/test/resources/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,2 @@
+Manifest-Version: 1.0
+Sling-Nodetypes: SLING-INF/nodetypes/nodetypes.cnd
```

Comment:
Interesting - I guess this makes sense because otherwise, you wouldn't be able to sufficiently mock our jcr since it would have no concept of our `pant:*` types. But I'm a bit surprised that this is the mechanism. Does the mocking code just know to check for this file existing at this location, or did you configure it somehow elsewhere and I just missed it?

Reply:
You are correct in your assumption about the node types.

The code grabs the manifest file from the classpath (in the appropriate place: META-INF/MANIFEST.MF) and uses it. I just had to place one in the test resources folder and have it refer to the right nodetypes file.